ansible/sepolia.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Verify Ansible versions
+  hosts: all
+  tags: always
+  become: false
+  run_once: true
+  gather_facts: false
+  tasks:
+    - local_action: command ./versioncheck.py
+      changed_when: false
+
+- name: Deploy Spolia Testnet nodes
+  become: true
+  serial: '{{ serial|default(1) }}'
+  hosts:
+    - nimbus-sepolia-metal
+  tasks:
+    - include_vars:
+        file: group_vars/nimbus-sepolia-metal.yml
```

Comment:
Wouldn't it be cleaner to move those to `vars` to make it clear those are loaded differently?

Reply:
Indeed, have created `vars` directory and moved `sepolia` and `kiln` files there.